topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -0,0 +1,444 @@
+>    {: .comment}
+>
+> 2. {% tool [Scanpy PlotTrajectory](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_plot_trajectory/scanpy_plot_trajectory/1.8.1+galaxy9) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in AnnData/Loom format"*: `output_h5ad` (output of **Scanpy PAGA** {% icon tool %})
+>    - *"Layout functions"*: `ForceAtlas2`
+>    - *"Location of legend"*: `On data`
+>    - *"Use programme defaults"*: {% icon history-share %} `No`
+>    - *"Name of cell annotation or gene that is used to color the nodes"*: `cell_type,ENSMUSG00000023274,ENSMUSG00000053977`
+>
+>
+>    > <comment-title> Choosing the layout </comment-title>
+>    >
+>    > We're going to pick the `ForceAtlas2` layout function for our PAGA plots as this is the same type of layout that we used for our FDG. 
+>    {: .comment}
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. How have the relationships between our cell clusters changed now?
+> 2. Which clusters are expressing our genes of interest, Cd4 and Cd8, at the highest levels?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The way the clusters are arranged has changed a bit now. The M4 cluster is right in the middle of the M1-3 clusters, rather than heading off on its own. The M1 cluster is looking like it is driving towards differentiation, which is not somthing we had necessarily been able to specify before by just looking at our cluster graphs or applying our biological knowledge.
+> > 2. Cd4 and Cd8 expression appears highest in the DP-L cluster. The expression of both Cd4 and Cd8 also appears higher than we might expect in the DP-M4 cluster - perhaps this is a sign that it is closer to the DP-L cluster than it seems in this simple plot. 
+> >
+> > ![DN cluster is on one side of the plot, with the DP L and T mature clusters on the other side. The other clusters are close together in the centre of the plot. Coloured to show higher Cd4 and Cd8 expression in the M4, DP L and T mature clusters.](../../images/scrna-case_trajectories/TrajectoriesPAGA.png "PAGA plots coloured by cell type, Cd4 expression, and Cd8 expression")
+> {: .solution}
+>
+{: .question}
+
+## Re-draw force-directed graph (again!)
+
+Force directed graphs can be initialised randomly, or we can prod it in the right direction. We'll prod it with our PAGA calculations. Note that you could also try prodding it with tSNE or UMAP. A lot of these tools can be used on top of each other or with each other in different ways, this tutorial is just one example. Similarly, you could be using any **obs** information for grouping, so could do this for *louvain* or *cell_type* for instance.
```

Comment:
(maybe expand tSNE and UMAP here too....? Or would that be overkill and we would expect users to at least roughly know what is meant by these terms?)